app/models/chat.rb

```diff
@@ -0,0 +1,5 @@
+class Chat < ApplicationRecord
+  belongs_to :user_a,  class_name: 'User', foreign_key: 'user_a_id'
```

Comment:
maybe, :user_receiver && :user_sender could be better names, WDYT?

Reply:
In the chat model, there is not a sender or receiver both of them send and receive messages.

Reply:
Agree with anto, there is no sender or receiver in the chat.